fiat-roles/src/main/java/com/netflix/spinnaker/fiat/permissions/RedisPermissionsRepository.java

```diff
@@ -129,6 +131,17 @@ public RedisPermissionsRepository put(@NonNull UserPermission permission) {
     return this;
   }
 
+  //I'm not satisfied with this.
+  //TODO: Find out a wait to fetch admin statuses in batches
+  private boolean isUserAdmin(String userID) {
+    try (Jedis jedis = jedisSource.getJedis()) {
+      Pipeline p = jedis.pipelined();
+      Response<String> admin = p.hget(userID, adminKey());
```

Comment:
If you're just doing 1 `hget`, you don't need a jedis `Pipeline`. You should just be able to `jedis.hget(...)`